contribs/drt/src/main/java/org/matsim/contrib/drt/analysis/DrtTripsAnalyser.java

```diff
@@ -410,27 +410,26 @@ public static String summarizeVehicles(Map<Id<Vehicle>, DrtVehicleDistanceStats.
 		format.setGroupingUsed(false);
 
 		DescriptiveStatistics driven = new DescriptiveStatistics();
-		DescriptiveStatistics revenue = new DescriptiveStatistics();
+		DescriptiveStatistics passengerMeter = new DescriptiveStatistics();
 		DescriptiveStatistics occupied = new DescriptiveStatistics();
 		DescriptiveStatistics empty = new DescriptiveStatistics();
 
 		for (DrtVehicleDistanceStats.VehicleState state : vehicleDistances.values()) {
 			driven.addValue(state.totalDistance);
-			revenue.addValue(state.totalRevenueDistance);
+			passengerMeter.addValue(state.totalPassengerMeter);
```

Comment:
As above: maybe `passengerDistance`? (consistency)

I am not against adding "meters" at the end, but then we need to update names of other variables/fields

Reply:
or "passengerTraveledDistance"